src/test/java/org/sqlflow/client/RunSQLFlowRemoteTest.java

```diff
@@ -17,6 +17,13 @@ public static void main(String[] args) {
       System.out.println("skip the test due to the SQLFlow client is not created");
       return;
     }
-    EnvironmentSpecificSQLFlowClient.hasGoodResponse(client, sqlProgram);
+
+    client.run(sqlProgram);
+    try {
+      client.release();
+    } catch (InterruptedException e) {
+      System.err.println("encounter an exception while releasing SQLFlow client");
```

Comment:
Print the error message of the exception?

Reply:
Sure. You are right.